# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

C42_ZmpWlkActionGoal action_goal
C42_ZmpWlkActionResult action_result
C42_ZmpWlkActionFeedback action_feedback
